src/main/java/core/basesyntax/Animal.java

```diff
@@ -2,10 +2,10 @@
 
 import java.util.Objects;
 
-public class Animal {
-    private String type;
-    private int age;
-    private long id;
+public abstract class Animal {
+    private final String type;
+    private final int age;
```

Comment:
why have we made fields final?
do we expect for example age filed to be immutable for ever?